### ISSUE TITLE
Decoupled Weight Decay Regularization

### ISSUE THREAD

**Issue by tannal** [tannal/ohmywork#12]
AdamW Keras Pytorch

https://arxiv.org/pdf/1711.05101.pdf
